db/migrate/20231027125416_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author_id, foreign_key: { to_table: :users }
+      t.references :post_id, foreign_key: { to_table: :posts }
```

Comment:
- [ ] Let's make sure the foreign keys (user) in the comments and likes tables (migration files) match the ERD. The only table to have `author_id` as foreign key is a posts table. This table should reference a user.